Penteract/Assets/Scripts/GameController.cpp

```diff
@@ -90,9 +91,11 @@ void GameController::Update() {
 
 	if (Input::GetKeyCodeDown(Input::KEYCODE::KEY_ESCAPE) || Input::GetControllerButtonDown(Input::SDL_CONTROLLER_BUTTON_START, 0)) {
 		if (isPaused) {
+			PauseController::SetIsPause(false);
```

Comment:
Shouldn't you take the script via the GET_SCRIPT macro? Correct me if I am wrong @Jordi-Gil

Reply:
It's because it's a static function and the GameController doesn't have the PauseController Game Object. This way, both scripts are "decoupled"